packages/logger/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "@rocket.chat/logger",
+	"version": "0.0.1",
+	"private": true,
+	"devDependencies": {
+		"@types/jest": "^27.4.1",
+		"eslint": "~8.45.0",
+		"jest": "~29.5.0",
+		"ts-jest": "~29.0.5",
+		"typescript": "~5.0.2"
```

Comment:
we're using TS 5.1 elsewhere.. can you pls update to 5.1 and also check if the other versions are also matching?